numba/npyufunc/ufuncbuilder.py

```diff
@@ -321,10 +321,14 @@ def build(self, cres):
         """
         # Buider wrapper for ufunc entry point
         signature = cres.signature
-        ptr, env, wrapper_name = build_gufunc_wrapper(self.py_func, cres,
-                                                      self.sin, self.sout,
-                                                      cache=self.cache)
-
+        info = build_gufunc_wrapper(
+            self.py_func, cres, self.sin, self.sout,
+            cache=self.cache, is_parfors=False,
+        )
+
+        wrapper_name = info.name
+        env = info.env
+        ptr = info.library.get_pointer_to_function(wrapper_name)
```

Comment:
```suggestion
        ptr = info.library.get_pointer_to_function(info.name)
```
and get rid of `wrapper_name = info.name` above ? don't think this is needed?